homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -81,15 +81,22 @@ def source_type(self) -> str:
     @property
     def extra_state_attributes(self) -> dict[str, any]:
         """Return the attributes."""
-        attrs = {
-            "mac": self._device.mac,
-            "ip_address": self._device.ip_address,
-        }
         if self._device.last_activity:
-            attrs["last_time_reachable"] = self._device.last_activity.isoformat(
-                timespec="seconds"
-            )
-        return attrs
+            return {
```

Comment:
Please make the return value consistent. Ie return a value explicitly in all cases.